Cairn.info.js

```diff
@@ -31,17 +31,25 @@
 */
 
 function detectWeb(doc,url) {
-	var xpath='//meta[@name="citation_journal_title"]';
-
-	if (ZU.xpath(doc, xpath).length > 0) {
+	breadcrumbPage = ZU.xpathText(doc, '//div[@id="breadcrump"]/a[last()]');
+	if (breadcrumbPage == "Ouvrage collectif") {
+		return "book";
+	} else if (breadcrumbPage == "Article") {
 		return "journalArticle";
+	} else if (breadcrumbPage == "Chapitre") {
+		return "bookSection";
+	} else if (breadcrumbPage == "Résumé") {
+		typeDocument = ZU.xpathText(doc, '//div[@id="breadcrump"]/a[2]');
+		if (typeDocument == "Revues") {
+			return "journalArticle";
+		} else if (typeDocument == "Ouvrages") {
+			return "book";
```

Comment:
I guess this should be "bookSection" e.g. your example http://www.cairn.info/jeu-d-echecs-comme-representation--9782728835904-page-111.htm